ds_judgements_public_ui/templates/pages/courts_and_tribunals.html

```diff
@@ -0,0 +1,36 @@
+{% extends "layouts/base.html" %}
+{% load i18n %}
+{% block robots %}
+  <meta name="robots" content="noindex,nofollow" />
+{% endblock robots %}
+{% block title %}
+  {% if context.page_title %}{{ context.page_title }} -{% endif %}
+  Judgements and decisions by court or tribunal
+{% endblock title %}
+{% block content %}
+  <div class="courts-and-tribunals container">
+    <div>
+      <div class="courts-and-tribunals__intro">
+        <h1>Judgments and decisions by court or tribunal</h1>
+        <p>
+          The UK courts and tribunal system is like a legal hierachy.  The Supreme Court is at the top, dealing with the final appeals.
```

Comment:
Is all text agreed?

Reply:
aha, I *think* this particular text is, but i will check with rose.